src/utils/i18n.ts

```diff
@@ -21,9 +21,10 @@ i18next
           provinceOrTerritory: "Province or Territory",
           capitalCity: "Capital City",
           attempts: "Attempts",
-          gamePotRoundInstruction: "TODO", // TODO: we probably don't need this
-          gameFlagRoundInstruction: "Select the flag",
-          gameCapitalRoundInstruction: "What is the capital",
+          gamePotRoundInstruction: "Select the Province of",
```

Comment:
I wouldn't add instructions for the initial map round. What do you think?

Reply:
The page does not display this now, but the option could be there. Let's say for A11y, where you need to read-out-loud.

Reply:
Good point!